app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/check_claim_summary.scala.html

```diff
@@ -82,14 +82,13 @@
     @summary(SummaryList(List(claimSummaryHelper.makeClaimTotalRow(claims))))
 
     @formWithCSRF(routes.EnterClaimController.checkClaimSummarySubmit, 'novalidate -> "novalidate") {
-        @pageHeading(messages(s"$key.are-duties-correct"), "govuk-heading-m govuk-!-margin-top-9", "h2")
         @radios(
             form = form,
             name = key,
             legend = messages(s"$key.are-duties-correct"),
             legendAsHeading = false,
             hintKey = None,
-            classes = "govuk-visually-hidden",
+            classes = "govuk-label--m govuk-!-margin-top-6",
```

Comment:
Using govuk-fieldset__legend govuk-fieldset__legend--m classes will be more maintainable in case there will be an update for fieldset component it will apply to these classes.